.github/workflows/send-telegram-notification.sh

```diff
@@ -1,131 +1,6 @@
-        sys.exit(1)
-    
-    # Get appropriate channel based on tag
-    chat_id, channel_type = get_channel_for_tag(tag)
-    
-    if not chat_id:
-        print(f"❌ No chat ID configured for {channel_type} channel.")
-        sys.exit(1)
-    
-    # Create message based on build status
-    status_mapping = {
-        "success": ("✅", "Completed Successfully"),
-        "failure": ("❌", "Failed"),
-        "cancelled": ("⏹️", "Cancelled"),
-        "skipped": ("⏭️", "Skipped")
-    }
-    
-    emoji, status_text = status_mapping.get(build_status, ("❓", "Unknown Status"))
-    
-    message = f"{emoji} *Android Build {status_text}*\n\n"
-    message += f"Tag: `{tag}`\n"
-    message += f"Status: {status_text}"
-    
-    if send_telegram_notification(chat_id, message, bot_key):
-        print(f"✅ Android build notification sent to {channel_type} chat {chat_id}")
-
-# -- Handles tag creation notifications -- #
-def handle_tag_notification():
-    bot_key = os.getenv("TELEGRAM_RELEASE_BOT_KEY")
-    dev_chat_id = os.getenv("TELEGRAM_DEV_RELEASE_CHAT_ID")
-    rc_chat_id = os.getenv("TELEGRAM_RC_RELEASE_CHAT_ID")
-
-    created_tags = os.getenv("CREATED_TAGS", "").split()
-    affected_branches = os.getenv("AFFECTED_BRANCHES", "").split()
-
-    if not bot_key: 
-        print("❌ TELEGRAM_RELEASE_BOT_KEY not found.")
-        sys.exit(1)
-
-    # Separate develop and RC tags
-    develop_tags = []
-    rc_tags = []
-    
-    for tag, branch in zip(created_tags, affected_branches):
-        if branch == 'develop':
-            develop_tags.append(tag)
-        else:
-            rc_tags.append(tag)
-
-    # Send notification for develop branch
-    if develop_tags and dev_chat_id: 
-        message = "🚀 *Development Build Started*\n\n"
-        message += "New versions being built:\n"
-        for tag in develop_tags: 
-            message += f"• `{tag}`\n"
-
-        if send_telegram_notification(dev_chat_id, message, bot_key):
-            print(f"✅ Notification sent to Open-mSupply dev Builds chat")
-    
-    # Send notification for RC branches
-    if rc_tags and rc_chat_id: 
-        message = "🏗️ *RC Build Started*\n\n"
-        message += "New versions being built:\n"
-        for tag in rc_tags: 
-            message += f"• `{tag}`\n"
-
-        if send_telegram_notification(rc_chat_id, message, bot_key):
-            print(f"✅ Notification sent to Open-mSupply RC Builds chat")
-
-def main(): 
-    notification_type = os.getenv("NOTIFICATION_TYPE", "tag_creation")
-
-    if notification_type == "android_build": 
-        handle_android_build_notification()
-    else:
-        handle_tag_notification()
-
-    
-if __name__ == "__main__":
-    main()
-EOF
\ No newline at end of file
+ARM64_FILENAME=$(ls ./client/packages/android/app/build/outputs/apk/arm64/release/);
+UNIVERSAL_FILENAME=$(ls ./client/packages/android/app/build/outputs/apk/arm64/release/);
+python3 ./send-telegram-notification.py
```

Comment:
@jmbrunskill noticed that the run failed. Error about path for this script. Thinking that it just have to be a direct path instead? `.github/workflows/send-telegram-notification.py`

Reply:
Yup thanks! Updated